### PR TITLE
228054 - column block video variation - excluding video cta action

### PR DESCRIPTION
**Pull request by VarsTheMars**
This PR doesn't contain the changes for the cta video action 

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:

- Before: https://main--mmsg-oly--aemsites.hlx.live/
- After: https://228054-column-video-variation--mmsg-oly--aemsites.hlx.live/

### CODE REVIEW

**Review by roohibengani — APPROVED** [review]

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -59,11 +59,14 @@
...
-      border-radius: 0.5rem;
-      max-width: 680px;
-      max-height: 400px;
-      aspect-ratio: 1.3;
+    .columns-img-col {
+      position: relative;
+
+      img {
+        border-radius: 0.5rem;
+        height: 280px;
```
Comment (mrugesh-adobe) [inline blocks/columns/columns.css:67]: this CSS will be affected to other variation of column block. @VarsTheMars

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -343,3 +346,24 @@
...
+}
+
+.section.video {
+  .columns-img-col::before {
+    content: url('../../icons/play-video-button.svg');
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white; /* change to desired color */
```
Comment (mrugesh-adobe) [inline blocks/columns/columns.css:366]: use variable here

**Review by mrugesh-adobe — CHANGES_REQUESTED** [review]

File: blocks/columns/columns.css (hunk 1 above)
Comment (VarsTheMars) [reply blocks/columns/columns.css]: the CSS changes has been reverted and updated @mrugesh-adobe

File: blocks/columns/columns.css (hunk 2 above)
Comment (VarsTheMars) [reply blocks/columns/columns.css]: added the global variable here

**Review by roohibengani — APPROVED** [review]

**Review by mrugesh-adobe — APPROVED** [review]

**Pull request merged**